scripts/exregional_process_radarref.sh

```diff
@@ -251,18 +253,18 @@ EOF
 #
 #-----------------------------------------------------------------------
 #
-exect="${EXECDIR}/process_NSSL_mosaic.exe"
+  exect="process_NSSL_mosaic.exe"
 
-if [ -f ${EXECDIR}/$exect ]; then
-  print_info_msg "$VERBOSE" "
-Copying the radar process  executable to the run directory..."
-  cp_vrfy ${EXECDIR}/${exect} ${WORKDIR}
-else
-  print_err_msg_exit "\
-The executable specified in GSI_exect does not exist:
-  exect = \"${EXECDIR}/$exect\"
-Build radar process and rerun."
-fi
+ if [ -f ${EXECDIR}/$exect ]; then
+   print_info_msg "$VERBOSE" "
+   Copying the radar process  executable to the run directory..."
+   cp_vrfy ${EXECDIR}/${exect} .
+ else
+   print_err_msg_exit "\
+   The executable specified in GSI_exect does not exist:
+   exect = \"${EXECDIR}/$exect\"
+   Build radar process and rerun."
+  fi
 #
```

Comment:
Need one more whitespace from line 258 to line 266.

Reply:
Done